test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewFeatures/DefaultDisplayTemplatesTest.cs

```diff
@@ -138,7 +138,7 @@ public void ObjectTemplate_IgnoresPropertiesWith_ScaffoldColumnFalse()
             var result = DefaultDisplayTemplates.ObjectTemplate(htmlHelper);
 
             // Assert
-            Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result));
+            Assert.Equal(expected, HtmlContentUtilities.HtmlContentToString(result), ignoreLineEndingDifferences: true);
```

Comment:
There's a lot of `Assert.Equal()` calls in this class and only one needs `ignoreLineEndingDifferences`? Why?